### DISCUSSION TITLE
在运行mcl.cmd时报错

### DISCUSSION THREAD

**Discussion by msj20080623** [mamoe/mirai#1422] [Q&A]
在运行mcl.cmd时报错：
'JAVA_BINARY' 不是内部或外部命令，也不是可运行的程序
或批处理文件。
'-jar' 不是内部或外部命令，也不是可运行的程序
或批处理文件。

**Comment by Karlatemp** [comment]
提供你的 `mcl.cmd` 文件内容

**Comment by benhuanggg** [comment]
你解决了吗，我和你是同样的情况，如果你解决了能告诉我怎么解决吗

**Reply by heipi666** [reply]
我解决了，路径有中文名，改成英文路径就好了
